test/fail_compilation/fail18372.d

```diff
@@ -0,0 +1,19 @@
+/*
+DISABLED: win32 win64 linux32 osx freebsd
+TEST_OUTPUT:
+---
+Error: `__va_list_tag` is not defined, perhaps `import core.stdc.stdarg;` ?
```

Comment:
How is this fixing the issue? The issue is about missing line number in the error message.

Reply:
Indeed I was so focused on the message I overlooked the line number.